consensus/hotstuff/integration/integration_test.go

```diff
@@ -40,6 +40,7 @@ func TestSingleInstance(t *testing.T) {
 }
 
 func TestThreeInstances(t *testing.T) {
+	t.Skip("flakey")
```

Comment:
We should use the flakey test skipping helper: https://github.com/onflow/flow-go/blob/cf38eb7aaf3a313ebe9337bd85b4a6582e852210/utils/unittest/unittest.go#L50